pyatlas/pyatlas/pyatlas_globalfunc.py

```diff
@@ -0,0 +1,10 @@
+"""
+This module contains global utility functions for pyatlas.
+"""
+
+
+def hello_atlas():
+    """
+    Print a welcome message!
+    """
+    print "Hello Atlas!"
```

Comment:
Is this just a default global function to be replaced in future?

Reply:
Good question :) It's pretty much just there for the `Getting Started` section of the README, which has the user call it to test that the package built correctly. In the future this module may have additional function definitions.